pages/api/content/index.js

```diff
@@ -1,5 +1,24 @@
 import sql from "~/lib/postgres";
 
-const handler = async (req, res) => res.status(200).json({ data: await sql`SELECT * FROM content;` });
+const handler = async (req, res) => {
+	if (req.method === "GET") {
+		return res.status(200).json({ data: await sql`SELECT * FROM content;` });
+	}
+	if (req.method === "POST") {
+		const { title, content_type, description, img_url, video_url } = req.body;
 
+		const checkContentTitle = await sql`SELECT * FROM content WHERE title=${title};`;
+
+		if (!title || !content_type || !description) {
+			return res.status(400).json({ message: "Please fill the required fields" });
+		}
```

Comment:
This will only have a very minor effect (undetectable really) but it might be beneficial to put this if block above line 10 (where you set the variable checkContentTitle. 

This block doesn't actually require any information from the first `await` so we could return more quickly by not waiting for it.

As I said - minor effect, doesn't need doing, just wanted to highlight it.